guides/plugins/plugins/plugin-fundamentals/logging.md

```diff
@@ -0,0 +1,92 @@
+# Logging
+
+## Overview
+
+As a plugin developer, you may want to log certain actions or errors to a log file to aid in debugging or to simply keep a record of performed actions.
+
+## Prerequisites
+
+This guide is built upon our [plugin base guide](../plugin-base-guide.md), which explains the basics of a plugin as a whole. Make sure to have a look at it to get started on building your first plugin.
+
+## Configuring Monolog
+
+First, you must make sure that your plugin loads package configuration from the `/Resources/config/packages` folder:
```

Comment:
Maybe add a link to the symfony docs, to also make clear that this boilerplate is not expected by us, but is plain symfony stuff?

Reply:
yep good idea